lib/find.js

```diff
@@ -11,6 +11,8 @@ const {getTimezoneAtSea, oceanZones} = require('./oceanUtils')
 
 let featureCache
 
+let featureFileFd = -1
```

Comment:
That does keep it more in line with how it worked before since it would have to load a specific file. I was thinking that keeping one file descriptor open doesn't hurt and is balanced out by the fact that we don't have to do an extra open() system call for each lookup.

I've reworked it locally to do it the way you've requested in case you feel strong about doing it that way. In that case, there's also a meet in the middle approach of a config option to manage this behavior. I was thinking for my use case I don't want preCache and I also don't want to do an extra open() for each lookup.

Reply:
I did a little bit of benchmarking and the performance is very similar. There seemed to be slightly more variation in runtime when opening for each loadFeatures() call but it was pretty small, around 5%.

I pushed the version that keeps re-opening the file to read from it